src/lib/setup/prompt.test.ts

```diff
@@ -76,6 +86,9 @@ describe("test prompt function", () => {
       az_sp_password: "a510c1ff-358c-4ed4-96c8-eb23f42bbc5b",
       az_sp_tenant: "72f988bf-86f1-41af-91ab-2d7cd011db47"
     });
+    jest.spyOn(inquirer, "prompt").mockResolvedValueOnce({
+      acr_name: "testACR"
+    });
     jest.spyOn(subscriptionService, "getSubscriptions").mockResolvedValueOnce([
```

Comment:
can you please elaborate on this? Do you want to select subscription Id first and then create service principal? Currently, we are doing the other way around, we can service principal and automatically figure out subscription id

Reply:
Here's the error I'm getting:
After az login the service principal gets created under the wrong tenant I believe when I select to generate a SP

`error:   Error: Get Token request returned http error: 400 and server response: {"error":"unauthorized_client","error_description":"AADSTS700016: Application with identifier '#' was not found in the directory '#'. This can happen if the application has not been installed by the administrator of the tenant or consented to by any user in the tenant. You may have sent your authentication request to the wrong tenant.\r\nTrace ID:#\r\nCorrelation ID: #r\nTimestamp: 2020-03-17 18:35:28Z","error_codes":[700016],"timestamp":"2020-03-17 18:35:28Z","trace_id":"#","correlation_id":"#","error_uri":"https://login.microsoftonline.com/error?code=700016"}`

Reply:
@andrebriggs how can this happen. We automatically create a service principal (which has Contributor role). How can service principal being created in the wrong tenant?

Reply:
@NathanielRose let's save that this for another PR. Good catch though